docs/quickstart.md

```diff
@@ -6,7 +6,7 @@ hidden: false
 ---
 
 <p align="center">
-  <img height="100" src="https://phylum.io/logo/dark-bckg.svg">
+  <img height="100" src="https://raw.githubusercontent.com/phylum-dev/cli/main/assets/dark-bckg.svg">
```

Comment:
Does this work?
```suggestion
  <img height="100" src="../assets/dark-bckg.svg">
```

Reply:
I am not confident this will work, will consider this after I get a chance to test the behavior on `readme.com`